workspace/utilities/lib/popup.xsl

```diff
@@ -131,11 +132,16 @@
 <xsl:template name="popup-close-button">
 	<xsl:param name="attr" />
 	<xsl:param name="url" />
-	<xsl:param name="content"/>
+	<xsl:param name="content">
+		<i class="block width-broader width-broad-from-xs width-thin-from-sm">
+			<xsl:call-template name="icon-close" />
```

Comment:
I like that this icon is the default for the popup-close-button but it should probably be commented out since it will crash as is (icon-close does not exist in the template at this moment).

Reply:
Why does it not exists ?

Reply:
Good question. A good basic icon collection should probably be in our template.

Reply:
`block width-broader width-broad-from-xs width-thin-from-sm`
Does all those classes exists in our basic theme config? @dominic-blain

Reply:
@dominic-blain 

> Good question. A good basic icon collection should probably be in our template.

Wanna send a PR please ?

Reply:
@dominic-blain we do have https://github.com/DeuxHuitHuit/symphonycms-template/blob/dev/workspace/utilities/lib/svg-icons.xsl

Reply:
@fhamon Yes they are all present by default.

Reply:
@nitriques I could propose a revamp of this file then in another PR including a close icon and leave this call-template. What do you think?

Reply:
@dominic-blain  I would prefer to merge you PR that includes the icon before merging this one in

Reply:
I Should be able to change the content. ilts not always an icon.

Reply:
Well @PascalPiche it is a parameter...

Reply:
@PascalPiche see a9392360dd